README.md

```diff
@@ -124,7 +124,7 @@ Depending on how ORT was installed, it can be run in the following ways:
 * If the Docker image was built, use
 
   ```shell
-  docker run ort --help
+  docker run ghcr.io/oss-review-toolkit/ort --help
```

Comment:
In line 109 there are the build instructions which use `ort` as the image `tag`, which was used by the previous version of the command. Do we need any alignment?

Reply:
Actually, the list item title in line 124 says "If the Docker image was built" (i.e. it was built locally by the user), so I believe the change to be invalid and be caused by a misunderstanding that these are not usage instructions for the case when a published image is downloaded from the Docker registry.

What do you think, @mnonnenmacher, as you where mentioning this stale PR in today's meeting?

Reply:
Yes, the sentence above does not make sense when using the published image. I think it would be best to just mention both possibilities, something like:

```
* To run the latest public Docker image, use

docker run ghcr.io/oss-review-toolkit/ort --help

* If the image was built locally, use

docker run ort --help

```